src/encoded/static/components/search.js

```diff
@@ -506,25 +506,30 @@ const ExperimentComponent = (props, reactContext) => {
                             <span>{result.assay_term_name}</span>
                         }
                         {(isFunctionalExperiment && referenceLoci.length > 0) ?
-                            <span>{referenceLoci.length === 1 ? ` of ${referenceLoci[0].symbol}` : ' of multiple loci'}</span>
+                            <span>{referenceLoci.length === 1 ? ` of ${referenceLoci[0].symbol} locus` : ' of multiple loci'}</span>
                         : null}
                         {result.biosample_ontology && result.biosample_ontology.term_name ? <span>{` in ${result.biosample_ontology.term_name}`}</span> : null}
-                        {(isFunctionalExperiment && examinedLoci.length > 0) ?
+                        {(isFunctionalExperiment && result.crispr_screen_readout) ?
                             <span>
-                                {` with readout of ${examinedLoci.join(', ')}`}
+                                {` with ${result.crispr_screen_readout}`}
                             </span>
                         : null}
-                        {(isFunctionalExperiment && result.crispr_screen_readout) ?
+                        {(isFunctionalExperiment && examinedLoci.length > 0) ?
                             <span>
-                                {` (${result.crispr_screen_readout})`}
+                                {` readout of ${examinedLoci.join(', ')}`}
                             </span>
                         : null}
                     </a>
-                    {result.biosample_summary ?
+                    {!isFunctionalExperiment && result.biosample_summary ?
                         <div className="result-item__highlight-row">
                             {result.biosample_summary ? <div><BiosampleSummaryDisplay summary={result.biosample_summary} organisms={organismNames.concat(GeneticModificationOrganismNames(biosamples))} /> </div> : null}
                         </div>
                     : null}
+                    {isFunctionalExperiment && (result.description || result.biosample_summary) ?
+                        <div className="result-item__highlight-row">
+                            {result.description ? <div>{result.description}</div> : result.biosample_summary ? <div><BiosampleSummaryDisplay summary={result.biosample_summary} organisms={organismNames.concat(GeneticModificationOrganismNames(biosamples))} /> </div> : null}
```

Comment:
I have a comment below for Series that applies similarly to this line.

Reply:
changed as suggested from comment below. ok